src/config.rs

```diff
@@ -253,6 +255,17 @@ pub enum PresetWidth {
 #[derive(knuffel::Decode, Debug, Clone, PartialEq)]
 pub struct DefaultColumnWidth(#[knuffel(children)] pub Vec<PresetWidth>);
 
+#[derive(knuffel::DecodeScalar, Debug, Default, PartialEq, Eq, Clone, Copy)]
+pub enum CenterColumnOnMove {
+    /// Moving to a column will keep not center the column.
+    #[default]
+    Never,
+    /// Moving to a column will always center the column.
+    Always,
+    /// Moving to a column will center the column if the column doesn't fit on the screen together with one of its neighbors.
+    NoFit,
+}
+
```

Comment:
I suspect if you run `cargo test` you will find out that you need to add this to the test below too, so please do that with a non-default value.